meshtastic/tests/test_powermon_stress.py

```
@@ -55,6 +65,15 @@ def test_power_stress_client_defaults_node_id_from_iface() -> None:
     assert client.node_id == 12345
 
 
+@pytest.mark.unit
+def test_power_stress_client_raises_when_iface_not_initialized() -> None:
+    """PowerStressClient should fail fast when myInfo.my_node_num is unavailable."""
+    iface = MagicMock()
+    iface.myInfo = None
+    with pytest.raises(ValueError, match="myInfo.my_node_num unavailable"):
```

Ruff (0.15.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)